### DISCUSSION TITLE
Unable to link Firebase in a Flutter Application

### DISCUSSION THREAD

**Discussion by sharonanderson-dev** [community/community#47066] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I don't know where the firebase project has been linked to my flutter app but after an hour of researching I found it, eventually when I tried to change the **google_service.json** there were no changes happening on my database it was still linked to my old one does anyone have any what's really happening over here?

**Comment by santosharron** [comment, marked as answer]
The **google-services.json** file is required for integrating Google services like Firebase into your Flutter app. If you have replaced the file but are still not seeing the changes, there are a few things you can check:

1. **Check the file path:** Make sure that the new `google-services.json` file is placed in the correct location in your Flutter project. Typically, it should be placed in the `android/app` directory of your project.
2. **Rebuild the project:** Try running `flutter clean` and then `flutter build` to ensure that the changes are properly applied.
3. **Verify that the new file is included in the build:** You can verify this by checking the log output of your build process to make sure that the new file is included in the build.
4. **Check the Firebase console:** Ensure that the right project is selected in the Firebase console and that the app is properly registered.

If you are still encountering issues, please provide more details such as the error message you are encountering, the exact steps you are taking, and the relevant parts of your code and configuration files. This will help me better understand and assist you with the issue.

**Reply by sharonanderson-dev** [reply]
Thanks for the Help! I had tried rebuilding the project with `flutter clean` and now it works pretty good,
